runtime/lang/menu_fi_fi.latin1.vim

```diff
@@ -473,6 +473,13 @@ fun! Do_toolbar_tmenu()
   tmenu ToolBar.FindHelp			Etsi ohjeesta
 endfun
 
+menutrans Toggle\ relati&ve\ Line\ Numbering<Tab>:set\ rnu!	Vaihda\ Suhteellinen\ rivinumerointi<Tab>:set\ nru!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap!		Vaihda\ Viivan\ k��re<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ word<Tab>:set\ lbr!		Vaihda\ K��ritty\ sanaan<Tab>:set\ lbr!
+menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et!			Vaihda\ V�lilehti\ laajenee<Tab>:set\ et!
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai!		Vaihda\ Automaattinen\ sisennys<Tab>:set\ ai!
+menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin!		Vaihda\ C-tyylinen\ sisennys<Tab>:set\ cin!
+menutrans &Show\ File\ Types\ in\ menu				N�yt�\ tiedostotyypit\ valikossa
```

Comment:
Is this a machine translation? The word salad here makes no sense... 
Toggle Line Wrapping is "Vaihda rivitys"
Toggle Wrapping at word is "Vaihda rivittäminen sanojen kohdalta"
Toggle Tab Expanding is "Vaihda sarkainten laajennusta"